.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+on: [pull_request]
+
+jobs:
+  ci:
+    name: Quality & Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [ '7.4', '8.0', '8.1' ]
+        symfony-version: [ '4.4.*', '5.4.*', '6.*' ]
+        composer-options: [ '', '--prefer-lowest' ]
+        exclude:
+          - php-version: '7.4'
+            symfony-version: '6.*'
```

Comment:
Why add them 2 lines before to exclude them here ? (or maybe I don't understand)

Reply:
That was more convenient to define a matrix with all options and removing them after.
The other option was to create an include with 2 lines (php8+sf6 & php8.1+sf6), but it was more verbose.